runtime/j9vm/javanextvmi.cpp

```diff
@@ -358,6 +358,9 @@ JVM_VirtualThreadMountEnd(JNIEnv *env, jobject thread, jboolean firstMount)
 				goto release2;
 			}
 
+			/* re-fetch as the memory allocation above may have moved the object */
```

Comment:
Please update the comment's formatting to match the surrounding comments.
```suggestion
			/* Re-fetch as the memory allocation above may have moved the object. */
```

Reply:
I was under the assumption that the objects won't move under VM access. Why is this an exception?